src/pages/managerDashboard.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Link from 'next/link';
+import NavBar from '@/components/Header/NavBar';
+import CustomizableAvatar from '@/components/common/customavatar/CustomizableAvatar';
+import CustomizableButton from '@/components/common/CustomizableButton';
+import FooterBar from '@/components/common/FooterBar';
+import styles from ''../styles/singleListing.module.css';
+
+const managerDashboard = () => {
+  return (
+    <div>managerDashboard</div>
+  )
+}
+
+export default managerDashboard
```

Comment:
Add a final new line here. 

Your 2 other PRs seem to have similar changes. Is there a mix-up or something?

Reply:
@jjleigh The last commit was to delete the commented out code I had in the NavBar module, I am looking at the other PRs to see what may have been accidentally committed elsewhere